contracts/interfaces/IUniswapV3Staker.sol

```diff
@@ -11,30 +17,51 @@ interface IUniswapV3Staker {
         uint32 claimDeadline,
         uint128 indexed totalReward
     );
+
+    /// @notice Event emitted when a liquidity mining incentive has ended
+    /// @param rewardToken The address of the token being distributed as a reward
+    /// @param pool The address of the Uniswap V3 pool
+    /// @param startTime The time when the incentive program begins
+    /// @param endTime The time when rewards stop accruing
     event IncentiveEnded(
         address indexed rewardToken,
         address indexed pool,
         uint32 startTime,
         uint32 endTime
     );
 
+    /// @notice Event emitted when a Uniswap V3 LP token has been deposited
+    /// @param tokenId The unique identifier of an Uniswap V3 LP token
+    /// @param owner The owner of the LP token
     event TokenDeposited(uint256 tokenId, address indexed owner);
+
+    /// @notice Event emitted when a Uniswap V3 LP token has been withdrawn
+    /// @param tokenId The unique identifier of an Uniswap V3 LP token
+    /// @param owner The address that the token will be withdawn to
     event TokenWithdrawn(uint256 tokenId, address to);
 
+    /// @notice Event emitted when a Uniswap V3 LP token has been staked
+    /// @param tokenId The unique identifier of an Uniswap V3 LP token
+    /// @param liquidity
```

Comment:
in periphery we just say:
```
@param liquidity The amount of liquidity for this position
```
it's kind of a loaded concept, I learn something new about it every day